app/poros/country_codes.rb

```diff
@@ -0,0 +1,258 @@
+      SHN: "Saint Helena, Ascension and Tristan da Cunha",
+      MAR: "Morocco",
+      MNG: "Mongolia",
+      NRU: "Nauru",
+      ISL: "Iceland",
+      LUX: "Luxembourg",
+      THA: "Thailand",
+      AND: "Andorra",
+      BGR: "Bulgaria",
+      NIC: "Nicaragua",
+      ITA: "Italy",
+      BLM: "Saint Barthélemy",
+      ASM: "American Samoa",
+      CIV: "Ivory Coast",
+      PLW: "Palau",
+      GUM: "Guam",
+      GIB: "Gibraltar",
+      AIA: "Anguilla",
+      COL: "Colombia",
+      TCD: "Chad",
+      BRA: "Brazil",
+      NPL: "Nepal",
+      IMN: "Isle of Man",
+      GUF: "French Guiana",
+      VCT: "Saint Vincent and the Grenadines",
+      PRK: "North Korea",
+      SXM: "Sint Maarten",
+      SUR: "Suriname",
+      KWT: "Kuwait",
+      SOM: "Somalia",
+      TUV: "Tuvalu",
+      BES: "Caribbean Netherlands",
+      COD: "DR Congo",
+      ALB: "Albania",
+      HND: "Honduras",
+      ECU: "Ecuador",
+      SDN: "Sudan",
+      FSM: "Micronesia",
+      HKG: "Hong Kong",
+      BEN: "Benin",
+      ESH: "Western Sahara",
+      LKA: "Sri Lanka",
+      BGD: "Bangladesh",
+      MDG: "Madagascar",
+      BHR: "Bahrain",
+      LIE: "Liechtenstein",
+      MKD: "North Macedonia",
+      CXR: "Christmas Island",
+      UKR: "Ukraine",
+      EGY: "Egypt",
+      ERI: "Eritrea",
+      MNP: "Northern Mariana Islands",
+      AZE: "Azerbaijan",
+      NER: "Niger",
+      LCA: "Saint Lucia",
+      MSR: "Montserrat",
+      VIR: "United States Virgin Islands",
+      SLE: "Sierra Leone",
+      DJI: "Djibouti",
+      HRV: "Croatia",
+      NOR: "Norway",
+      DMA: "Dominica",
+      URY: "Uruguay",
+      GMB: "Gambia",
+      MDA: "Moldova",
+      AFG: "Afghanistan",
+      PRT: "Portugal",
+      MNE: "Montenegro",
+      BRN: "Brunei",
+      FJI: "Fiji",
+      MEX: "Mexico",
+      CHE: "Switzerland",
+      RWA: "Rwanda",
+      CYP: "Cyprus",
+      BMU: "Bermuda",
+      TWN: "Taiwan",
+      JPN: "Japan",
+      QAT: "Qatar",
+      DNK: "Denmark",
+      CCK: "Cocos (Keeling) Islands",
+      BVT: "Bouvet Island",
+      BWA: "Botswana",
+      MCO: "Monaco",
+      LBN: "Lebanon",
+      LBY: "Libya",
+      MYS: "Malaysia",
+      VAT: "Vatican City",
+      CRI: "Costa Rica",
+      ESP: "Spain",
+      MTQ: "Martinique",
+      SVK: "Slovakia",
+      BIH: "Bosnia and Herzegovina",
+      BDI: "Burundi",
+      CPV: "Cape Verde",
+      OMN: "Oman",
+      BLR: "Belarus",
+      MAF: "Saint Martin",
+      BRB: "Barbados",
+      UNK: "Kosovo",
+      TZA: "Tanzania",
+      WSM: "Samoa",
+      ETH: "Ethiopia",
+      IND: "India",
+      EST: "Estonia",
+      IRL: "Ireland",
+      MDV: "Maldives",
+      SYC: "Seychelles",
+      UGA: "Uganda",
+      FLK: "Falkland Islands",
+      STP: "São Tomé and Príncipe",
+      NIU: "Niue",
+      TLS: "Timor-Leste",
+      IRQ: "Iraq",
+      KNA: "Saint Kitts and Nevis",
+      CUW: "Curaçao",
+      NFK: "Norfolk Island",
+      MHL: "Marshall Islands",
```

Comment:
Great job overall, later on in the refactor part we should try to find out a way to simplify this instead of writing them all out like this. It would make our code look cleaner.